### DISCUSSION TITLE
Fx 110 none-sense shadow missing under non-selected, inactive tabs

### DISCUSSION THREAD

**Discussion by MikeVapour** [Aris-t2/CustomCSSforFx#550] [General]
Hi,

I applied the _Fx 110 none-sense - remove line below active tab_ to FF110, but as I use very light-colored tabs, it becomes apparent that the line/shadow below the inactive tabs is gone too.
Is there any way to just remove the line from the active tab but keep it for the remaining tabs?

I enclose the image to illustrate what I am after (the line is present to the right of the selected tab, but missing to the left from the selected tab:

![image](https://user-images.githubusercontent.com/99497780/213924535-2eab75f1-f5b1-4df9-8d5c-8f9b18335b02.png)

**Comment by Aris-t2** [comment]
By default there is no line below active tab, if using this projects files, is it?

![image](https://user-images.githubusercontent.com/6013569/213932801-90640d85-b9eb-4144-921a-2010e47d44b1.png)

**Reply by MikeVapour** [reply]
That is correct. The active tab is fine.
The inactive tab is wrong: it does not have the line separating it from the toolbar. You see it clearly in light colored themes. Like here:
![image](https://user-images.githubusercontent.com/99497780/213933643-5afb997e-e8d8-4deb-be84-60f67391dc5b.png)
The _France 24_ is the inactive tab in this image. It is FF 110.

This is what it should look like (and it does look ok in FF109). Again, the France 24 tab is inactive, and here it has the line (shadow) at the bottom.
![image](https://user-images.githubusercontent.com/99497780/213933763-4bd36717-7e03-4176-a5e9-a396419c3aeb.png)

**Reply by Aris-t2** [reply]
How does this look without themes or with other themes?

Firefox 110 probably uses themes colors differently compared to pre-110 versions.

**Reply by Aris-t2** [reply]
I maybe found something, that can improve this on Fx 110+ by adding a minor line at the bottom of inactive tabs simulating the mentioned border. 

The line is created by the a shadow an can be used inside to `classic_squared_tabs.css` file.

```css
#main-window:not(:-moz-lwtheme) #TabsToolbar #tabbrowser-tabs .tabbrowser-tab:not([selected]) .tab-stack .tab-content {
  box-shadow: inset 0 -1px 0 0 var(--classic_squared_tabs-border2) !important;
}
```

It already is in 4.2.9v5 test build.

**Reply by MikeVapour** [reply]
yes! this works real well! Thank you very much! :)
